wolfcrypt/src/aes.c

```diff
@@ -7879,7 +7883,7 @@ int wc_AesCcmEncrypt(Aes* aes, byte* out, const byte* in, word32 inSz,
     const word32 wordSz = (word32)sizeof(word32);
 
     /* sanity check on arguments */
-    if (aes == NULL || out == NULL || in == NULL || nonce == NULL
+    if (aes == NULL || (inSz != 0 && (in == NULL || out == NULL)) || nonce == NULL
```

Comment:
80 chars

Reply:
fixed.  searched for others -- plenty but no more new ones.